projects/indicators/src/modules/text-highlight/text-highlight.directive.ts

```diff
@@ -30,12 +30,32 @@ export class SkyTextHighlightDirective
    * Specifies the text to highlight.
    */
   @Input()
-  public skyHighlight: string;
+  public set skyHighlight(value: string | string[]) {
+    value = value || [];
+    this._skyHighlight = [];
+    /* istanbul ignore else */
+    if (typeof value === 'string') {
+      this._skyHighlight = [value as string];
+    } else if (Array.isArray(value)) {
+      for (let i = 0; i < value.length; i++) {
```

Comment:
I just realized this could be done with a simple `filter()` call rather than creating an array and pushing items to it in a loop. This would also mean you can get rid of the assignment to a new array on line 35 since both branches of the `if` statement will now assign the array.

```
this._skyHighlight = value.filter(item => !!item);
```